posts/2022-09-21-history-maker-projects.adoc

```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: "Open Liberty- A brief history in maker projects of a cloud-native Java runtime
+title: "Open Liberty- A brief history in maker projects of a cloud-native Java runtime"
 # Do NOT change the categories section
 categories: blog
 author_picture: https://avatars3.githubusercontent.com/lauracowen
```

Comment:
Could you update the picture link to point at `https://avatars3.githubusercontent.com/u/3322532`